chart/kubeapps/Chart.yaml

```diff
@@ -33,4 +33,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.6.0
+version: 7.6.1
```

Comment:
Not a big deal, but shouldn't this be `7.6.1-dev0` so that it's ahead of the released 7.6.1 chart but before any release 7.6.2 chart? Fine as is, just remembering our previous conv about it.

Reply:
Ah, that's true... since there are no actual changes in the chart (apart from the container versions), I forgot to update it. In fact, I'd say it should be `v7.6.2-dev0`, because the `-foo` suffix is for prereleases (`semver compare v7.6.1-dev0 v7.6.1` yields `-1`